frontend/src/summary.pug

```diff
@@ -56,8 +57,11 @@
         a(v-on:click="resetDateRange") Reset date range
       .summary-picker__checkboxes.summary-picker__section
         label.filter-breakdown
-          input.mui-checkbox(type="checkbox", v-model="filterBreakdown",
-            v-on:change="setSummaryHash()")
```

Comment:
Is `setSummaryHash` contained in getFiltered so there is no need to call it again?

Reply:
![image](https://user-images.githubusercontent.com/16630400/109967319-3b308280-7d2c-11eb-9844-2db92c74c012.png)
Yes